tests/utilities/test_data/loop_test_data.csv

```diff
@@ -0,0 +1,6 @@
+# Metadata string
```

Comment:
therefore this file can be removed

Reply:
oops this only makes sense when you've all the comments :smile:

Reply:
yes.